src/common/Header/styled.js

```diff
@@ -0,0 +1,147 @@
+import styled from 'styled-components';
+import { ReactComponent as VideoIcon } from "./VideoIcon.svg";
+import { ReactComponent as SearchIcon } from "./SearchIcon.svg";
+
+export const Wrapper = styled.header`
+    background: ${({ theme }) => theme.color.woodsmoke};
+    height: 94px;
+    display: grid;
+    gap: 80px;
+    grid-template-columns: 1.5fr 1fr 2fr;
+    padding: 35px 16px 35px 16px;
+    align-content: center;
+    color: ${({ theme }) => theme.color.white};
+    width: 100%;
+
+    @media(max-width: ${({ theme }) => theme.breakpoint.mobileMax}px){
+        height: 142px;
+        grid-template-columns: 1fr 1fr;
+        padding: 16px;
+        gap: 19px;
+    }
+`;
+
+export const TitleWrapper = styled.div`
+    display: flex;
+    gap: 12px;
+    align-items: center;
+    justify-self: right;
+
+    @media(max-width: ${({ theme }) => theme.breakpoint.mobileMax}px){
+        gap: 8px;
+        margin-top: 8px;
+        justify-self: left;
+    }
+`;
+
+export const StyledVideoIcon = styled(VideoIcon)`
+    @media(max-width: ${({ theme }) => theme.breakpoint.mobileMax}px){
+        width: 17px;
+        height: 17px;
+    }
+`;
+
+export const TitleText = styled.div`
```

Comment:
I suggested using clamp, but now I think fixed `font-size: 24px;` widh media query would be better.
We also should use `font-family: 'Poppins', sans-serif;` in GlobalStyle.js so we don't repeat the code 

![image](https://user-images.githubusercontent.com/117176251/220740083-48d74bd2-7492-40d7-887d-9ee82e39e984.png)